homeassistant/components/energy/websocket_api.py

```diff
@@ -35,6 +39,26 @@ def async_setup(hass: HomeAssistant) -> None:
     websocket_api.async_register_command(hass, ws_get_prefs)
     websocket_api.async_register_command(hass, ws_save_prefs)
     websocket_api.async_register_command(hass, ws_info)
+    websocket_api.async_register_command(hass, ws_solar_forecast)
+
+
+@singleton("energy_platforms")
+async def async_get_energy_platforms(hass: HomeAssistant) -> dict[str, Any]:
```

Comment:
We should type the `async_get_solar_forecast` item in the dict return value since that function signature is part of the energy API.